pkg/apis/numaflow/v1alpha1/vertex_types.go

```diff
@@ -384,6 +384,10 @@ type VertexLimits struct {
 	// Read batch size
 	// +optional
 	ReadBatchSize *uint64 `json:"readBatchSize,omitempty" protobuf:"varint,1,opt,name=readBatchSize"`
+	// Read timeout seconds
+	// +kubebuilder:default=1
+	// +optional
+	ReadTimeoutSeconds *uint32 `json:"readTimeoutSeconds,omitempty" protobuf:"varint,3,opt,name=readTimeoutSeconds"`
```

Comment:
Protobuf index should be 2.

BTW - I have a feeling that it might be better to use `readTimeout` directly with `time.Duration`, since it makes sense ppl want to configure it in `ms` level, WDYT?

Reply:
I thought about duration but only think parsing error.  let me change it